docs/README.md

```diff
@@ -3,17 +3,24 @@
 ![Official logo](img/Box86Logo.png "Official Logo")
 Linux Userspace x86 Emulator with a twist
 
-Box86 lets you run x86 Linux programs (such as games) on non-x86 Linux, like ARM (host system needs to be 32bit little-endian).
+----
+
+Compiling/Installation
+----
+> Compilation instructions can be found [here](COMPILE.md)  
+> For an easy 1-click installation, check out box86 on [pi-apps](https://github.com/Botspot/pi-apps/)
```

Comment:
Isn't pi-apps only for PI? In that case, I would prefer "On Raspeberry Pi, checkout out pi-apps for an easy..." or something similar.